recipes/trafaret/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "trafaret" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "f8f27c4a11054b64ae4a6fac091dfd57e59b7a19ef3294b6eb552b376b1da8eb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - trafaret
+    - trafaret.extras
+    - trafaret.utils
+    - trafaret.visitor
+    - trafaret.constructor
+
+about:
+  home: https://github.com/Deepwalker/trafaret
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Ultimate transformation library that supports validation, contexts and aiohttp'
+
+  description: |
+    Trafaret is rigid and powerful lib to work with foreign data, configs etc.
+    It provides simple way to check anything, and convert it accordingly to
+    your needs.
+  doc_url: http://trafaret.readthedocs.io/en/latest/
+  dev_url: https://github.com/Deepwalker/trafaret
+
+extra:
+  recipe-maintainers:
+    - elatt
+    - dsakagi
+    - peterhurford
```

Comment:
@peterhuford are you ok with being a maintainer?

Reply:
Yes